schemas/app-sre/maintenance-1.yml

```diff
@@ -82,6 +88,7 @@ properties:
           type: boolean
       required:
       - provider
+      - page
```

Comment:
For my understanding, this is a new required field that will be used to actually notify subscribers (list them or use status page API of some sort), right?

it sounds weird to have it required with the provider pattern, but as we have only maintenance announcements of type statuspage for now, it's fine of course.

Reply:
this is exactly the provider pattern. we are under `oneOf`: https://github.com/app-sre/qontract-schemas/pull/647/files#diff-8954fe83fbfc14d023e6ab1d5e0eca9b03c919e4d0dfbcb2d2eec677360c0a3bR67